data/technology/social_networking_websites.json

```diff
@@ -148,6 +148,7 @@
     "Qzone",
     "Raptr",
     "Ravelry",
+    "reddit",
```

Comment:
```suggestion
    "Reddit",
```

Reply:
Their wordmark and website title both use "reddit", but their brand guidelines use "Reddit".

Reply:
Yeah, the logo is lowercase, but they use "Reddit" in prose, which is the context Corpora is likely to be used.

(Interesting about the title, Google shows "reddit: the front page of the internet" from desktop and "Reddit" on mobile.)

Anyway, other than the logo, I don't see any "reddit" at, for example, these pages:

https://www.reddithelp.com/en
https://www.redditinc.com/
https://www.redditinc.com/policies
https://redditblog.com/